src/lib/storage/table.cpp

```diff
@@ -126,8 +126,10 @@ void Table::_validate_existing_definition(const std::string& name, const std::st
   Assert(!_is_instantiated.at(pos), "A column with the given name was already added!");
 }
 
-void emplace_chunk(Chunk chunk) {
-  // Implementation goes here
+void Table::emplace_chunk(Chunk chunk) {
```

Comment:
The expected behaviour should be following:
`Adds a chunk to the table. If the first chunk is empty, it is replaced.` <-- table.hpp

Reply:
Good point - we missed that.